onnxscript/type_annotation.py

```diff
@@ -179,7 +185,7 @@ def pytype_to_type_strings(pytype: TypeAnnotationValue) -> list[str]:
     if isinstance(pytype, typing.TypeVar):
         constraints = pytype.__constraints__
         if constraints:
-            return pytype_to_type_strings(Union.__getitem__(constraints))
+            return pytype_to_type_strings(Union[constraints])
```

Comment:
Suggested by linter, looks .. strange

Reply:
Not sure if this will fail in some python versions?

Reply:
Curious which linter suggested this?

Reply:
PYLINT, I think I'll revert and skip this

Reply:
```
  Advice (PYLINT) C2801
    Unnecessarily calls dunder method __getitem__. Access item via subscript.
    (unnecessary-dunder-call)
    See [unnecessary-dunder-call](https://pylint.pycqa.org/en/latest/
    user_guide/messages/convention/unnecessary-dunder-call.html).

    To disable, use `  # pylint: disable=unnecessary-dunder-call`

        185  |    if isinstance(pytype, typing.TypeVar):
        186  |        constraints = pytype.__constraints__
        187  |        if constraints:
    >>> 188  |            return pytype_to_type_strings(Union.__getitem__(constraints))
        189  |        bound = pytype.__bound__
        190  |        if bound is None:
        191  |            return list(ALL_TENSOR_TYPE_STRINGS)
```

Reply:
Hmm. The CI test failure with ort-nightly is unrelated right? Saw that on main too. I thought it was due to this change.

Reply:
Yes it’s being fixed in the version bump PR